### PR TITLE
Add agent version API to agent reference

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds agent version API cURL example to agent reference

## Motivation and Context
agent version API added for 5.18

### CODE REVIEW

File: content/sensu-go/5.18/reference/agent.md [hunk 1]
```diff
@@ -623,12 +623,20 @@ sc.exe query SensuAgent
 
 ### Get service version
 
-To get the current agent version using the `sensu-agent` tool:
+There are two ways to get the current agent version: the `sensu-agent` tool and the agent version API.
+
+To get the current agent version with the `sensu-agent` tool:
```
Comment (cwjohnston) [inline content/sensu-go/5.18/reference/agent.md]:
```suggestion
To get the version of the current `sensu-agent` tool:
```

File: content/sensu-go/5.18/reference/agent.md [hunk 2]
```diff
@@ -623,12 +623,20 @@ sc.exe query SensuAgent
...
 
 {{< highlight shell >}}
 sensu-agent version
 {{< /highlight >}}
 
+To get the current agent version with the agent version API:
```
Comment (cwjohnston) [inline content/sensu-go/5.18/reference/agent.md]:
```suggestion
To get the version of the running `sensu-agent` service:
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
The two methods serve slightly different purposes. I hope these suggestions make the difference more clear.

**Comment by hillaryfraley** [comment]
@cwjohnston yes, thank you! An important distinction.

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**